### ISSUE TITLE
Void context in block not detected if block only contains one line

### ISSUE THREAD

**Issue by untitaker** [bbatsov/rubocop#5551]
Consider this code:

```ruby
array.each do |_item|
  42
end
```

--------

## Expected behavior

Rubocop should emit a `Lint/Void` for the second line.

## Actual behavior

Rubocop considers this code valid.

## RuboCop version

master, or 0.52

## Notes

The following code is linted as expected, until the second int literal is removed:

```ruby
array.each do |_item|
  42
  42
end
```

**Comment by pocke** [comment]
```ruby
array.each do |_item|
  42
end

array.each do |_item|
  42
  42
end
```

```bash
$ ruby-parse test.rb
(begin
  (block
    (send
      (send nil :array) :each)
    (args
      (procarg0 :_item))
    (int 42))
  (block
    (send
      (send nil :array) :each)
    (args
      (procarg0 :_item))
    (begin
      (int 42)
      (int 42))))
```

When a block body has one sentence, it does not have a `begin` node. The cop's checks `begin` and `kwbegin` node only, so the cop can't catch it now.
https://github.com/bbatsov/rubocop/blob/031b29b495ad435dc69172fb7603042894faae53/lib/rubocop/cop/lint/void.rb#L82-L85
